### PR TITLE
Snowflake dialect support

### PR DESCRIPTION
**Pull request by alex-odysseus**
Addresses https://github.com/OHDSI/WebAPI/issues/2069

It can be merged only after releasing of the corresponding libraries:
- DatabaseConnector (@schuemie please support to merge https://github.com/odysseusinc/DatabaseConnector/tree/snowflake into https://github.com/OHDSI/DatabaseConnector/tree/develop and release after)
- SqlRender (@schuemie please support to merge https://github.com/odysseusinc/SqlRender/tree/snowflake into https://github.com/OHDSI/SqlRender/tree/develop and release after)
- ACHILLES (@fdefalco please support to merge https://github.com/OHDSI/Achilles/tree/snowflake into https://github.com/OHDSI/Achilles/tree/develop and release after)
- CommonDataModel (@clairblacketer please support to merge https://github.com/odysseusinc/CommonDataModel/tree/snowflake into https://github.com/OHDSI/CommonDataModel/tree/main)
- ARACHNE Commons (https://github.com/OHDSI/ArachneCommons/tree/snowflake_webapi to be merged to https://github.com/OHDSI/ArachneCommons/tree/1.17.x and released after)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
     <circe.version>1.9.4</circe.version>
     <jersey.version>2.14</jersey.version>
-    <SqlRender.version>1.9.0</SqlRender.version>
+    <SqlRender.version>1.9.1-SNAPSHOT</SqlRender.version>
```
Comment (anthonysena) [inline pom.xml:32]: Please update to v1.10.0 (just released)

File: pom.xml [hunk 2]
```diff
@@ -193,7 +193,7 @@
     <server.ssl.key-password></server.ssl.key-password>
     <server.servlet.context-path>/WebAPI</server.servlet.context-path>
 
-    <arachne.version>1.17.2</arachne.version>
+    <arachne.version>1.17.3-SNAPSHOT</arachne.version>
```
Comment (alex-odysseus) [inline pom.xml:196]: 1.17.3

File: pom.xml [hunk 3]
```diff
@@ -1769,6 +1769,20 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>webapi-snowflake</id>
+      <properties>
+        <snowflake.enabled>true</snowflake.enabled>
+        <snowflake.driver.version>3.13.14</snowflake.driver.version>
```
Comment (alex-odysseus) [inline pom.xml:1776]: 3.13.22

**Comment by chrisknoll** [comment]
Also, please merge with master and resolve conflicts.

**Comment by alex-odysseus** [comment]
Closing as been replaced by https://github.com/OHDSI/WebAPI/pull/2096

**Pull request closed without merging**